spec/requests/rack_attack_spec.rb

```diff
@@ -32,23 +32,18 @@ def till_throttled(requests_to_send:, throttled_status: 429, mode: :default, &bl
 
     case mode
     when :slow
-      sleep_time = 0.2
-      time_scale = 1.0
+      time_advance = 1
     when :default
-      sleep_time = 0
-      time_scale = 0.0001
+      time_advance = 0.01
     else
       raise 'unknown mode'
     end
 
-    # travel to hour boundary so we always start at 00:00
-    Timecop.travel((Time.current + 1.hour).beginning_of_hour)
-    # adjust speed of time to reliably trigger burst throttling (10req/1sec)
-    Timecop.scale(time_scale)
```

Comment:
replace scale with manually stepping through time with freeze for greater reliability